params/protocol_params.go

```diff
@@ -166,15 +166,13 @@ const (
 	RefundQuotient        uint64 = 2
 	RefundQuotientEIP3529 uint64 = 5
 
-	BlobTxBytesPerFieldElement         = 32      // Size in bytes of a field element
-	BlobTxFieldElementsPerBlob         = 4096    // Number of field elements stored in a single data blob
-	BlobTxBlobGasPerBlob               = 1 << 17 // Gas consumption of a single data blob (== blob byte size)
-	BlobTxMinBlobGasprice              = 1       // Minimum gas price for data blobs
-	BlobTxBlobGaspriceUpdateFraction   = 3338477 // Controls the maximum rate of change for blob gas price
-	BlobTxPointEvaluationPrecompileGas = 50000   // Gas price for the point evaluation precompile.
-
-	BlobTxTargetBlobGasPerBlock = 3 * BlobTxBlobGasPerBlob // Target consumable blob gas for data blobs per block (for 1559-like pricing)
-	MaxBlobGasPerBlock          = 6 * BlobTxBlobGasPerBlob // Maximum consumable blob gas for data blobs per block
+	BlobTxBytesPerFieldElement             = 32      // Size in bytes of a field element
+	BlobTxFieldElementsPerBlob             = 4096    // Number of field elements stored in a single data blob
+	BlobTxBlobGasPerBlob                   = 1 << 17 // Gas consumption of a single data blob (== blob byte size)
+	BlobTxMinBlobGasprice                  = 1       // Minimum gas price for data blobs
+	BlobTxBlobGaspriceUpdateFractionCancun = 3338477 // Controls the maximum rate of change for blob gas price
```

Comment:
What about making the update fraction variables private since they are only used in the default objects in the same module?

Reply:
Or even remove them?

Reply:
I don't see much issue exporting them? They're big magic numbers in the protocol, people should probably be able to access them if they want? We also export `DefaultBaseFeeChangeDenominator` and `DefaultElasticityMultiplier` for 1559?

Reply:
I think it doesn't make sense to expose them since it can lead to weird bugs like the one you just fixed. It is not safe to rely on these constants in any code since the limits are configurable arbitrarily. This is unlike 1559, which is not configurable.

Reply:
I'll remove them then